Atomic/syscontainers.py

```diff
@@ -1061,13 +1053,23 @@ def _get_containers_at(self, checkouts, are_preinstalled, containers=None):
             if fullpath.endswith(".0") or fullpath.endswith(".1"):
                 continue
 
-            with open(os.path.join(fullpath, "info"), "r") as info_file:
+            try:
+                fd = open(os.path.join(fullpath, "info"), "r")
+            except IOError:
+                # Something got wrong?
+                continue
+            with fd as info_file:
```

Comment:
please drop this part, if something went wrong we must report the error, or am I missing something?